security/ssl-tls/lets-encrypt/index.en.md

```diff
@@ -23,8 +23,16 @@ To avoid duplicates, alwaysdata blocks Let's Encrypt certificate generation for
 The creation of these certificates is dependent on DNS propagation: the address must point to alwaysdata servers. Once the address is added in **Web > Sites**, the system will attempt to generate its certificate every 30 minutes for 24 hours. This will then change to once a day.
 {{% /notice %}}
 
+{{% notice note %}}
+Let's Encrypt implemented a limit on the number of SSL certificates created per domain. When the twentieth SSL certificate is to be created - that is, the twentieth subdomain has been added to **Web > Sites** - the system will generate a wildcard certificate to manage the following subdomains.
```

Comment:
`Let's Encrypt put a rate limit on the number of SSL certificates. From 20 subdomains, i.e. from the 20th subdomain added to **Web > Sites**, the system will generate`...